src/component/admin/apiToken/ApiTokenForm/SelectProjectInput.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { FC, useState } from 'react';
+import { Checkbox, FormControlLabel, TextField, Box } from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import { IAutocompleteBoxOption } from 'component/common/AutocompleteBox/AutocompleteBox';
+
+const wildcard = '*';
```

Comment:
We usually call wildcards like this either ALL_PROJECTS or ALL_ENVIRONMENTS, I'm not as familiar with the front end as I am the backend but I'm hoping there's already a constant for this! If not, we could either make one or I would suggest renaming this to ALL_PROJECTS instead